src/common/client-app.ts

```diff
@@ -35,6 +35,7 @@ export function start(boardVM: IBoardVM, config?: AppConfiguration) {
 
 export class App {
   shadow: model.Board = { };
+  severShadow: model.Board = { };

```

Comment:
I know that naming is hard, but maybe you could find better names for `serverShadow`, `shadow` and `shadowDifferences` to help to understand the real meaning behind them.